components/forms/RadioButton/react.test.js

```diff
@@ -28,10 +28,65 @@ describe('RadioButton', () => {
     })
   })
 
-  describe('with user actions', () => {
-    it('should call onChange on click', () => {
-      wrapper.find('input').simulate('click')
-      expect(props.onChange).toHaveBeenCalledTimes(1)
+  describe('Initial values for the radio component when it is created', () => {
+    it('Should render with the default checked as undefined', () => {
```

Comment:
I think all the defaultProp tests can be a bit more automatic by exporting `Radio.defaultProps` (rewrite to `export const defaultProps = {}` and `Radio.defaultProps = defaultProps`), this way we don't have to update tests and defaultProps when we decide to change a default option.

Reply:
Tarik and I discussed this in person. This can be exported/imported into the test case, and would help to make test cases a bit more automatic.